packages/core/stories/flow-layout.doc.stories.mdx

```diff
@@ -3,33 +3,33 @@ import { FlowLayout } from "@salt-ds/core";
 import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
 
 <Meta
-  title="Documentation/Core/Layout/FlowLayout"
+  title="Documentation/Core/Layout/Flow Layout"
   component={FlowLayout}
   parameters={{
     viewMode: "docs",
   }}
 />
 
-# Flow layout
+# Flow Layout
```

Comment:
not sure if this should be with a capital to match the component name? @sion-docs

Reply:
I was thinking if we were to change here, to find and change all references of the layouts to use capitals, since it's component names?

Reply:
That would be preferable

Reply:
will do

Reply:
This may need to be revisited, we (the content team) need to come up with specific guidelines for component naming.